test/e2e/api/matchMedia/map-projection.html

```diff
@@ -0,0 +1,93 @@
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta name="color-scheme" content="dark light">
+    <title>Map Projection Toggle</title>
+    <script type="module" src="mapml.js"></script>
+    <style media="">
+        body {
+            display: flex;
+        }
+
+        mapml-viewer {
+            flex: 2;
+            height: 50vh;
+        }
+
+        .description {
+            padding: 20px;
+            flex: 1;
+        }
+
+        @media (max-width: 768px) {
+            body {
+                flex-direction: column;
+            }
+        }
+    </style>
+</head>
+
+<body>
+    <script>
+        document.addEventListener('DOMContentLoaded', () => {
+            const map = document.querySelector('mapml-viewer');
+            let projectionMatcher = map.matchMedia('(map-projection: OSMTILE)'),
+                osmtile = map.querySelector('#OSMTILE'),
+                cbmtile = map.querySelector('#CBMTILE');
+
+            const hideLayer = (matchesOSMTILE) => {
+                map.whenLayersReady().then(() => {
+                    if (matchesOSMTILE) {
+                        cbmtile.setAttribute('hidden', 'hidden');
+                        osmtile.removeAttribute('hidden');
+                    } else {
+                        osmtile.setAttribute('hidden', 'hidden');
+                        cbmtile.removeAttribute('hidden');
+                    }
+                });
+            };
+            hideLayer(projectionMatcher.matches);
+            document.querySelector('.switchOSM').addEventListener('click', () => map.projection = 'OSMTILE');
+            document.querySelector('.switchCBMT').addEventListener('click', () => map.projection = 'CBMTILE');
+
+            const changeHandler = (e) => {
+                let matchesOSMTILE = projectionMatcher.matches;
+                hideLayer(matchesOSMTILE);
+            };
+            projectionMatcher.addEventListener('change', changeHandler);
+        });
+    </script>
+
+    <mapml-viewer projection="OSMTILE" zoom="9" lat="44.630550504861795" lon="-103.86611938476564" controls>
+        <map-layer id="OSMTILE" label="OpenStreetMap" checked="">
+            <map-link rel="license" title="Â© OpenStreetMap contributors CC BY-SA"
+                href="https://www.openstreetmap.org/copyright"></map-link>
+            <map-extent units="OSMTILE" checked="checked">
+                <map-input name="z" type="zoom" value="18" min="0" max="18"></map-input>
+                <map-input name="x" type="location" units="tilematrix" axis="column" min="0" max="262144"></map-input>
+                <map-input name="y" type="location" units="tilematrix" axis="row" min="0" max="262144"></map-input>
+                <map-link rel="tile" tref="https://tile.openstreetmap.org/{z}/{x}/{y}.png"></map-link>
+            </map-extent>
+        </map-layer>
+        <map-layer media="(map-projection: CBMTILE)" id="CBMTILE" checked>
```

Comment:
I think we should take the media attribute off here, it may be confusing to future us